lnbits/core/templates/core/index.html

```diff
@@ -172,6 +172,17 @@ <h5 class="q-my-md">{{SITE_TAGLINE}}</h5>
             </a>
           </div>
         </div>
+        <div class="row">
+          <div class="col">
+            <a href="https://getalby.com">
```

Comment:
hey! can you please add a `target="_blank"` to all the hrefs? i think they should all have one.

Reply:
I think we should do it in another PR. Maybe even create a component call `a-external` that not only adds `target="_blank"` but also `rel="noopener noreferrer"` for ancient browsers.

Reply:
ok